src/contracts/BConst.sol

```diff
@@ -17,7 +17,7 @@ contract BConst {
   /// @notice The minimum swap fee that can be set.
   uint256 public constant MIN_FEE = BONE / 10 ** 6;
   /// @notice The maximum swap fee that can be set.
-  uint256 public constant MAX_FEE = BONE / 10;
+  uint256 public constant MAX_FEE = BONE - MIN_FEE;
```

Comment:
why set it to `BONE - MIN_FEE` instead of just `BONE`? It doesn't seem to trigger any errors, which tells me that either

- it's perfectly lawful
- we're not testing for high fee values because of other pre-existing `vm.assume` or `bound`

I'm currently trying to make sure it's not the latter

Reply:
i think it was chosen in order to avoid edge cases, which now that i'm looking at the BMath tree it reverts in every swap operation (because of the `calcSpotPrice` division by zero), but it doesn't revert in `joinswap` or `exitswap`, causing an unconsistent behaviour

because of this, i believe is better to provide a highly taxed normal behaviour than fall into unchecked reverting cases, i think now is better 99.9999% instead of allowing 100%